src/data/mock-response.js

```diff
@@ -0,0 +1,64 @@
+export const mockResponse = {
+  type: 'div',
+  label: 'j3860704z-utah_phillips_one_2018.mp3',
+  items: [
+    {
+      type: 'div',
+      label: 'First segment',
+      items: [
+        {
+          type: 'div',
+          label: 'Sub-Segment 1.1',
+          items: []
+        },
+        {
+          type: 'span',
+          label: 'Segment 1.1',
+          begin: '00:00:03.32',
+          end: '00:00:10.32'
+        },
+        {
+          type: 'span',
+          label: 'Segment 1.2',
+          begin: '00:00:11.23',
+          end: '00:09:03.23'
+        }
+      ]
+    },
+    {
+      type: 'div',
+      label: 'Second segment',
+      items: [
+        {
+          type: 'div',
+          label: 'Sub-Segment 2.1',
+          items: [
+            {
+              type: 'div',
+              label: 'Sub-Segment 2.1.1',
+              items: []
+            },
+            {
+              type: 'span',
+              label: 'Segment 2.1',
+              begin: '00:09:03.24',
+              end: '00:15:00.00'
+            }
+          ]
+        }
+      ]
+    },
+    {
+      type: 'div',
+      label: 'Excellent Third Segment Series',
+      items: [
+        {
+          type: 'span',
+          label: 'Spannn',
+          begin: '00:15:01.00',
+          end: '00:21:00.00'
+        }
+      ]
+    }
+  ]
+};
```

Comment:
@adamjarling Seems like this is not used anywhere :) Or does this have a purpose in the future?

Reply:
So I created that because I was offline for a bit, and needed access to an offline data object to keep working.  Is it ok if we leave it in for now, in case we're offline and want to pull in a local file?

Reply:
Yeah, that'd be good.